.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: envattrs release workflow
+
+on:
+  push:
+    tags:
+      - '*'
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release:
+    name: Release to PyPI
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.6
+      - run: |
+          curl -sSL https://install.python-poetry.org | python - \
+        env:
+          POETRY_VERSION: 1.1.15
```

Comment:
This is pretty cool, I'm pretty sure that the env clause applies to the command ran, but reads kind of in reverse, which is a bit 🤷🏻

Reply:
I also prefer the env to come before run, but it's really minor.